test/tests.js

```diff
@@ -1 +1,30 @@
-import './example.test.js';
+import compareNumbers from '../compareNumbers.js';
+
+const test = QUnit.test;
+
+test('tests', function(assert) {
+    //Arrange
+    // Set up your parameters and expectations
+    const expected = 5;
+    const guess1 = 2;
+    const guess2 = 5;
+    const guess3 = 7;
+    
+    //Act 
+    // Call the function you're testing and set the result to a const
+    const test1 = compareNumbers(guess1, expected);
+    const test2 = compareNumbers(guess2, expected);
+    const test3 = compareNumbers(guess3, expected);
+    
+
+
+    //Assert
+    // Make assertions about what is expected valid result
+    assert.equal(test1, -1);
+    assert.equal(test2, 0);
+    assert.equal(test3, 1);
+    assert.throws(
```

Comment:
I see that you're throwing an error here, but I'm not sure you're actually testing your function. Also, it would be good if you had more than one expected value just to be sure you're getting maximal test coverage.